### ISSUE TITLE
[a11y] - focus order of element - ccordinate input / coordinate search

### ISSUE THREAD

**Issue by transkamp** [open-pioneer/trails-openlayers-base-packages#429]
### Issue description

open: https://open-pioneer.github.io/trails-demo/openlayers-base-packages/dev/samples/showcase/?demo=coordinateSearch

as a screen reader user, i first need to know for which coordinate system i can enter valid coordinates. to do this, i need to know which coordinate system has already been selected. therefore, it is better to display the srs dropdown list first in the tab order and then the input field.

i also need information as a screen reader user that i can enter my input by pressing the enter key on the input field. alternatively, the form should have a submit button.

Violation of https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html

![Image](https://github.com/user-attachments/assets/72b51904-695b-4712-b706-8c64c2b7345a)

### Validations

- [x] Read and follow our [code of conduct](https://github.com/open-pioneer/.github/blob/main/CODE_OF_CONDUCT.md)
- [x] Read the [contributing guidelines](https://github.com/open-pioneer/trails-starter/blob/main/docs/Contributing.md)
- [x] Check that there isn't already a similar issue
